src/main/java/g2001_2100/s2011_final_value_of_variable_after_performing_operations/readme.md

```diff
@@ -0,0 +1,73 @@
+2011\. Final Value of Variable After Performing Operations
+
+Easy
+
+There is a programming language with only **four** operations and **one** variable `X`:
+
+* `++X` and `X++` **increments** the value of the variable `X` by `1`.
+* `--X` and `X--` **decrements** the value of the variable `X` by `1`.
+
+Initially, the value of `X` is `0`.
+
+Given an array of strings `operations` containing a list of operations, return the final value of `X`  *after performing all the operations* .
```

Comment:
You may use generatemd tool to create readme files.

https://github.com/javadev/LeetCode-in-Java/releases/tag/generatemd

https://github.com/javadev/LeetCode-in-Java/releases/download/generatemd/generatemd-1.0.jar